cmd/helm/template_test.go

```diff
@@ -175,6 +175,13 @@ func TestTemplateCmd(t *testing.T) {
 			expectKey:   "subchart1/templates/service.yaml",
 			expectValue: "kube-version/major: \"1\"\n    kube-version/minor: \"6\"\n    kube-version/gitversion: \"v1.6.0\"",
 		},
+		{
+			name:        "check_kube_api_versions",
+			desc:        "verify --api-version overrides kubernetes api versions",
+			args:        []string{subchart1ChartPath, "--api-version", "helm.k8s.io/test"},
```

Comment:
This flag needs to be updated with the new plural flag

Reply:
oops - fixed!